shared/checks/oval/package_gnome-shell_installed.xml

```diff
@@ -0,0 +1,25 @@
+<def-group>
```

Comment:
This file should not exist and be generated with templates.

Reply:
In addition, this is a dependency of gdm, so this actually should be package_gdm_installed.